src/libraries/Microsoft.PowerFx.Core/Types/TypeSymbolTable/PrimitiveTypesSymbolTable.cs

```diff
@@ -61,6 +61,18 @@ internal override bool TryGetTypeName(FormulaType type, out string typeName)
             typeName = _knownTypes.Where(kvp => kvp.Value.Equals(type)).FirstOrDefault().Key;
 
             return typeName != null;
+        }
+
+        bool INameResolver.LookupType(DName name, out FormulaType fType) 

```

Comment:
>LookupType [](http://example.com/codeflow?start=27&length=10)

curious - we're trying to avoid (ideally remove ) PrimitiveTypesSymbolTable , what forced us to add this?

Reply:
ah , just added an implementation of the method that makes sense. could just throw an exception or return defaults . it is not used